scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -151,6 +151,26 @@ <h4 class="panel-title">
       <span class="glyphicon glyphicon-comment"></span>
     </a>
   {% endif %}
+
+
+  {% if variant.panels|count > 0 %}
```

Comment:
```suggestion
  {% if form.variant_type.data == 'research' %}
```

Reply:
In Lund we're not really using the research/clinical feature of Scout, we're always loading all variants. Thus, this suggestion won't really work for us (since we're never in "research mode").

I see that it becomes cluttered when every variant has the badge, but it still has some value to be able to quickly see what panels the gene is in without clicking on the variant (at least this is what our users are after).

Maybe one could consider  a different, less intrusive way of showing this?

Reply:
I see.. You may want to reconsider this at some point as you start amassing whole genomes, but let's try to keep things compatible. How about adding it to the gene hover? The gene hover currently only shows a full name / description for the gene, and the panel info might be rather logical there without increasing clutter?